commons-dlp/src/main/java/org/exoplatform/commons/dlp/rest/DlpRestServices.java

```diff
@@ -0,0 +1,88 @@
+package org.exoplatform.commons.dlp.rest;
+
+import io.swagger.annotations.*;
```

Comment:
```suggestion
```

Reply:
It's necessary this import to use swagger annotations (@API, @ApiOperation ..)